tests/Fixer/Alias/EregToPregFixerTest.php

```diff
@@ -33,72 +33,100 @@ public function testFix(string $expected, ?string $input = null): void
         $this->doTest($expected, $input);
     }
 
-    public function provideFixCases(): array
+    public function provideFixCases(): iterable
     {
-        return [
```

Comment:
very hard to see if anything changed here, in terms or test cases, or only if it's change of returning array to returning yeilds

Reply:
Disabling whitespace diff on GitHub helps too.